src/main/resources/changesets/changelog-consent-86.0.xml

```diff
@@ -0,0 +1,15 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="86.0" author="connorlbark">
+        <!--Add mapping and dataset property type columns -->
+        <addColumn tableName="datasetproperty">
+            <column name="property_type" type="varchar(255)"/>
+            <column name="schema_property" type="varchar(255)"/>
+        </addColumn>
+
+        <sql>
+            UPDATE datasetproperty SET property_type='number' WHERE propertyKey=5
```

Comment:
I think we can also add the string type for all of the other property key values

Reply:
good point - added!